packages/middleware/src/createServer.ts

```diff
@@ -50,6 +51,8 @@ async function createServer<
 
   const app = express();
 
+  configureFileUpload(app, options.fileUpload);
```

Comment:
(nitpick, feel free to skip this comment) imo we could improve readiness, by changing this line to sth like this:

```
  app.use(createMulterMiddleware(options.fileUpload))
```

as it's clear in most of the file which middlewares we register: `cookieParser`, `jsonBodyParser` etc. `configureFileUpload(app)` hides this informtion, and we have to navigate to separate file to get to know that multer is used under the hood.